### PR TITLE
Feat/navbar modal

### PR DESCRIPTION
**Pull request by keshavkumar-in**
Added Navbar and Modal Component.
Restructured whole CL structure to fix JS issue. In the earlier structure JS was not working.

### CODE REVIEW

File: docs/components/alert/index.html [hunk 1]
```diff
@@ -1,36 +1,188 @@
...
+              <button class="button button-primary button-round" role="button">
+                GitHub
+              </button>
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container">
+      <aside class="sidebar">
```
Comment (hsnice16) [inline docs/components/alert/index.html:42]: Nice, you have used semantic HTML.

File: lib/package/components/navbar.css [hunk 2]
```diff
@@ -2,9 +2,12 @@
 
 /* Navbar Components */
 .navbar {
+  /* position: sticky;
```
Comment (hsnice16) [inline lib/package/components/navbar.css]: You can remove these commented codes if they do not need.

**Review by hsnice16 — APPROVED** [review]
Nice work. Good naming convention, and keep that consistent that's awesome. Great work. Just added some suggestions.

File: docs/components/alert/index.html [hunk 3]
```diff
@@ -1,36 +1,188 @@
...
-    sandbox="allow-scripts allow-same-origin"
-  >
-  </iframe>
-</div>
+  <body>
+    <header class="navbar">
+      <nav class="nav">
+        <div class="navbar-brand">
+          <a href="/" class="navbar-logo">
+            <img src="/assets/images/WrixbyUI.png" alt="" />
```
Comment (vrx29) [inline docs/components/alert/index.html:20]: please put some data in alt tag

File: lib/package/components/navbar.css (hunk 2 above)
Comment (keshavkumar-in) [reply lib/package/components/navbar.css]: > You can remove these commented codes if they do not need.

Yeah I forgot. Thanks for noting

File: docs/components/alert/index.html (hunk 3 above)
Comment (Dev-AkshatGupta) [inline docs/components/alert/index.html:20]: Suggestion Please also mention something in alt tags too

**Review by Dev-AkshatGupta — APPROVED** [review]
Else everything looks cool to me

File: docs/components/alert/index.html (hunk 3 above)
Comment (keshavkumar-in) [reply docs/components/alert/index.html:20]: Yeah! Thanks for pointing

File: docs/components/alert/index.html [hunk 4]
```diff
@@ -1,36 +1,188 @@
...
+          <li class="sidebar-item">
+            <a href="/docs/components/slider/" class="sidebar-item-link">
+              Slider
+            </a>
+          </li>
+        </ul>
+      </aside>
+
+      <div class="component-section padding-horizontal-3">
+        <div class="component-section-background"></div>
```
Comment (vrx29) [inline docs/components/alert/index.html:132]: What's the use of this div? If its not used please remove it

File: lib/package/components/navbar.css [hunk 5]
```diff
@@ -2,9 +2,12 @@
...
+  top: 0;
+  z-index: 1; */
```
Comment (vrx29) [inline lib/package/components/navbar.css]: You can remove commented code

**Review by vrx29 — APPROVED** [review]

File: docs/components/alert/index.html (hunk 4 above)
Comment (keshavkumar-in) [reply docs/components/alert/index.html:132]: This is the background of the content section with slighly blur effect

**Pull request merged**